index.html

```diff
@@ -60,5 +65,159 @@ <h2>What to Expect at a Hackathon</h2>
             <img src="hackathon_infomatic.png" alt="An infographic hackathon layout">
         </div>
         
+        <h2>Why they are important:</h2>
+        <p>As developers, we are natural lifelong learners. This learning is nourished by 
+            collaborations with other people to find new ways to solve tasks.
+            Hackathons are great to learn by meeting people, discovering new technologies, and challenging yourself!
+        </p>
+        <br>
+        <p>Where to find them:</p>
+
+        <ul style="list-style-type: circle;">
```

Comment:
All of the ul element style properties needs to be listed in the style section not in the html.